daisy_workflows/export/disk_export_ext.wf.json

```diff
@@ -18,9 +18,9 @@
       "Value": "projects/compute-image-tools/global/images/family/debian-9-worker",
       "Description": "image to use for the exporter instance"
     },
-    "export_instance_disk_size": {
+    "buffer_disk_size": {
       "Value": "200",
-      "Description": "size of the export instances disk, this disk is unused for the export but a larger size increase PD read speed"
+      "Description": "size of the export instances buffer disk, this disk starts from a larger size to increase PD read speed, and grows on demand"
```

Comment:
'larger' is a comparative word, consider using the real size here in description